common/src/main/java/net/whimxiqal/journey/data/versionmanager/MysqlDataVersionManager.java

```diff
@@ -0,0 +1,45 @@
+package net.whimxiqal.journey.data.versionmanager;
+
+import net.whimxiqal.journey.Journey;
+import net.whimxiqal.journey.data.DataVersion;
+import net.whimxiqal.journey.data.sql.mysql.MySqlConnectionController;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static net.whimxiqal.journey.data.DataManagerImpl.VERSION_TABLE_NAME;
+
+public class MysqlDataVersionManager extends SqlDataVersionManager {
+    public MysqlDataVersionManager(MySqlConnectionController controller) {
+        super(controller);
+    }
+
+    @Override
+    public DataVersion version() {
+        // We don't have to check for a version file here since it was never used while MySQL existed, so we should leave it for SQLite to check.
+
+        try (Connection connection = controller.establishConnection()) {
+            PreparedStatement statement = connection.prepareStatement(String.format("SHOW TABLES LIKE '%s'", VERSION_TABLE_NAME));
+
+            ResultSet tableList = statement.executeQuery();
+            if (!tableList.next()) return DataVersion.V000; // The table does not exist, so probably version 0
+
+            PreparedStatement readStatement = connection.prepareStatement(String.format("SELECT Max(db_version) as DBVersion FROM %s;",VERSION_TABLE_NAME));
```

Comment:
Use the designated static variables for `db_version` and `journey_db_version` down below